src/extra/fromEvent.ts

```diff
@@ -29,7 +29,9 @@ export class NodeEventProducer implements InternalProducer<any> {
   constructor(private node: EventEmitter, private eventName: string) { }
 
   _start(out: InternalListener<any>) {
-    this.listener = (e: any) => out._n(e);
+    this.listener = (...args: Array<any>) => {
+        return (args.length > 1) ? out._n([...args]) : out._n(args[0]);
```

Comment:
This will also be good information to have in the future, do you prefer amended/rebased stuff for changes like this, or smaller fix commits?